apps/mobile/src/components/Feed/Socialize/ProcessedCommentText.tsx

```diff
@@ -0,0 +1,61 @@
+import { ReactNode, useCallback, useMemo, useRef } from 'react';
+import { Text } from 'react-native';
+
+import { GalleryBottomSheetModalType } from '~/components/GalleryBottomSheet/GalleryBottomSheetModal';
+import { Typography } from '~/components/Typography';
+import { VALID_URL } from '~/shared/utils/regex';
+
+import { WarningLinkBottomSheet } from '../Posts/WarningLinkBottomSheet';
+
+type LinkProps = {
+  url: string;
+};
+
+const LinkComponent = ({ url }: LinkProps) => {
+  const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
+  const handleLinkPress = useCallback(() => {
+    bottomSheetRef.current?.present();
+  }, []);
+
+  return (
+    <>
+      <Text className="text-shadow" onPress={handleLinkPress}>
+        {url}
+      </Text>
```

Comment:
Should we wrap this with `TouchableOpacity` or  `Pressable`?

Reply:
unfortunately when I use either it breaks the wrapping and I wasn't able to find tailwind classes that would make it behave like Text :(

I figured since we don't need to show any interaction effect in this case (since we immediately show the bottom sheet), the trade off is worth it

without pressable/touchable opacity
<img width="391" alt="CleanShot 2023-10-05 at 21 04 16@2x" src="https://github.com/gallery-so/gallery/assets/80802871/1a030555-0e18-4000-9993-4fb51999948c">

with it
<img width="394" alt="CleanShot 2023-10-05 at 21 04 22@2x" src="https://github.com/gallery-so/gallery/assets/80802871/89c529a3-3a89-4cac-96cb-fd0636771a00">

Reply:
good catch though